sdk/storage/azure-storage-datamovement/inc/azure/storage/datamovement/blob_folder.hpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+
+#include <azure/storage/blobs.hpp>
+
+namespace Azure { namespace Storage { namespace DataMovement {
+
+  class BlobFolder final {
```

Comment:
missing docs for this file.
For each public method and for the file in the top right

Reply:
I'll add docs later